spec/controllers/social_networking/comments_controller_spec.rb

```diff
@@ -17,14 +19,13 @@ module SocialNetworking
         participant: participant,
         text: "I like cheeses",
         item_id: 5,
-        item_type: "SocialNetworking::OnTheMindStatement")
+        item_type: "SocialNetworking::OnTheMindStatement"
```

Comment:
Given your advice in the past, I would consider `SocialNetworking::OnTheMindStatement.to_s`